nest-forms-backend/src/nases/dtos/xml.dto.ts

```diff
@@ -0,0 +1,14 @@
+export type XmlObject = {
```

Comment:
I would name it sth like `NasesAttachementXmlObject`, this is a bit too generic.